tests/acceptance/test_django.py

```diff
@@ -0,0 +1,26 @@
+import pathlib
+
+from django.db import connection
+
+
+def test_django_migrations_run_properly(db):
```

Comment:
I know the `db` fixture comes from pytest-django, but I'm afraid this could be really unclear. Would you mind adding a very explicit alias:

```python
# In this file of conftest.py
def django_db(db):
	# db is provided by pytest-django
    return db
```